### PR TITLE
ENCD-5057-add-experiment-audit-for-no-processed-data

### PR DESCRIPTION
**Pull request by jnau**

### CODE REVIEW

File: src/encoded/audit/experiment.py [hunk 1]
```diff
@@ -3990,6 +3990,23 @@ def audit_experiment_eclip_queried_RNP_size_range(value, system, excluded_types)
...
+
+    if not raw_data:
+        return
+
+    if not processed_data:
+        detail = 'Experiment {} only has raw data and does not contain any processed data'.format(audit_link(path_to_text(value['@id']), value['@id']))
+        yield AuditFailure('lacking processed data', detail, level='WARNING')
+    else:
+        return
+
```
Comment (gabdank) [inline src/encoded/audit/experiment.py]: You don't need else return, it is the end of the function anyway

File: src/encoded/audit/functional_characterization_experiment.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        value.get('original_files'), excluded_files)
+
+    for function_name in function_dispatcher_with_files.keys():
+        yield from function_dispatcher_with_files[function_name](value, system, files_structure)
+
+    excluded_types = excluded_files + ['deleted', 'replaced']
+    for function_name in function_dispatcher_without_files.keys():
+        yield from function_dispatcher_without_files[function_name](value, system, excluded_types)
+
+    return
```
Comment (gabdank) [inline src/encoded/audit/functional_characterization_experiment.py]: I don't think we need this return statement

File: src/encoded/tests/test_audit_experiment.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import pytest, re
 
+
```
Comment (gabdank) [inline src/encoded/tests/test_audit_experiment.py]: I don't think you need to add a new line here

File: src/encoded/audit/functional_characterization_experiment.py (hunk 2 above)
Comment (jnau) [reply src/encoded/audit/functional_characterization_experiment.py]: src/encoded/audit/experiment.py also has this return statement. should i remove it there or ignore it?

**Comment by hitz** [comment]
branch has conflicts.

**Comment by hitz** [comment]
Merging it without new demo, only changes were to test code; tests pass.

**Pull request merged**